src/sourmash/tax/tax_utils.py

```diff
@@ -1703,25 +1903,25 @@ def __post_init__(self):
 
     def set_status(self, query_info, containment_threshold=None, ani_threshold=None):
         # if any matches, use 'below_threshold' as default; set 'match' if meets threshold
-        if any([containment_threshold, ani_threshold]):
+        if any([containment_threshold is not None, ani_threshold is not None]):
             self.status="below_threshold"
         self.set_query_ani(query_info=query_info)
-        if ani_threshold:  # if provided, just use ani thresh, don't use containment threshold
+        if ani_threshold is not None:  # if provided, just use ani thresh, don't use containment threshold
             if self.query_ani_at_rank >= ani_threshold:
                 self.status = 'match'
         # should we switch to using weighted here? I think yes, but this would be behavior change
-        elif containment_threshold and self.fraction >= containment_threshold:
-        #elif containment_threshold and self.f_weighted_at_rank >= containment_threshold:
+        elif containment_threshold is not None and self.fraction >= containment_threshold:
+        #elif containment_threshold is not None and self.f_weighted_at_rank >= containment_threshold:
```

Comment:
remove?

Reply:
modified - at some point, I'd like to chat about potentially using weighted results instead of unique results for ordering ..